Kinvey/Kinvey/FindOperation.swift

```diff
@@ -192,7 +201,61 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, AnyRandomAccessCo
                             cache.save(entities: entities)
                         }
                     }
-                    completionHandler?(.success(entities))
+                    if self.autoPagination,
+                        entities.count == IntMax(MaxSizePerResultSet)
```

Comment:
This solution is interesting, but I think it's a little simpler to make the `_count` call first in all cases when `autoPagination` is set. The .NET SDK does it this way.